src/bundle/Resources/public/js/scripts/admin.contenttype.matrix.js

```diff
@@ -49,8 +49,8 @@
         updateDisabledState(parentNode);
 
         parentNode.querySelectorAll(SELECTOR_COLUMN_CHECKBOX).forEach((btn) => {
```

Comment:
I know this has already been written before, but the name `btn` doesn't seem to fit here :)

Reply:
true :D